pyquil/tests/test_operator_estimation.py

```diff
@@ -837,9 +837,9 @@ def test_measure_observables_inherit_noise_errors(forest):
 PRAGMA ADD-KRAUS H 2 "(0.9486832980505138 0.0 0.0 0.9486832980505138)"
 PRAGMA ADD-KRAUS H 2 "(0.0 0.31622776601683794 0.31622776601683794 0.0)"
 '''
-    assert calibr_prog1 == Program(expected_prog)
-    assert calibr_prog2 == Program(expected_prog)
-    assert calibr_prog3 == Program(expected_prog)
+    assert calibr_prog1.out() == Program(expected_prog).out()
```

Comment:
Why this change?

Reply:
These tests were broken and the CI apparently isn't spinning up qvm/quilc correctly so they're not getting hit.

Reply:
The operator estimation tests add like 10 minutes to the CI, so probably deserves a larger discussion.

Reply:
I'm confused - adding a `.out()` is just changing to string equality vs. program equality. Was the latter what was causing 10 minutes delay?

Reply:
Ok so I misremembered -- here is the real story. Some of the operator estimation tests are broken, but are not being caught by CI. I began attempting to fix this one, but when doing program equality the diff was useless, so I changed it to string equality to see if I could understand better what was going on. Ultimately, I decided that I would need to spend more time than I wanted to atm to get to the bottom of the issue, which is unrelated to this PR. So, if you'd like me to revert the change, I am happy to. As for the 10 minutes, I was saying that these skipped tests, when they are un-skipped, add a lot of time to the build.